src/test/java/com/google/devtools/build/lib/packages/util/BazelMockCcSupport.java

```diff
@@ -74,6 +73,49 @@ public void setup(MockToolsConfig config) throws IOException {
     config.append(
         TestConstants.TOOLS_REPOSITORY_SCRATCH + "tools/cpp/BUILD",
         "alias(name='host_xcodes',actual='@local_config_xcode//:host_xcodes')");
+    if (config.isRealFileSystem() && shouldUseRealFileSystemCrosstool()) {
+      config.append(
+          TestConstants.TOOLS_REPOSITORY_SCRATCH + "tools/cpp/BUILD",
+          """
+          toolchain_type(name = 'toolchain_type')
```

Comment:
```suggestion
          toolchain_type(name = 'toolchain_type')
          filegroup(name = "current_cc_toolchain")
```

@iancha1992 This should fix the test failure. In Bazel 8, C++ rules still have a hardcoded dep on this target.

Reply:
@fmeum I've added the line. But there's still failures.